test/integrations/sources/cordial/data.ts

```diff
@@ -0,0 +1,678 @@
+                bmID: '',
+                first: 0,
+                properties: {
+                  category: 'Shirts',
+                  url: 'http://example.com/shirts',
+                  description: 'A really cool khaki shirt.',
+                  price: 9.99,
+                  title: 'Khaki Shirt',
+                  test_key: 'value',
+                },
+              },
+            },
+            source: {},
+          },
+        ],
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+      pathSuffix: '',
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            output: {
+              batch: [
+                {
+                  context: {
+                    library: {
+                      name: 'unknown',
+                      version: 'unknown',
+                    },
+                    integration: {
+                      name: 'Cordial',
+                    },
+                    traits: {
+                      userId: '6690fe3655e334xx028xxx',
+                      email: 'jondoe@example.com',
+                      _id: '6690fe3655e334xx028xxx',
+                      channels: {
+                        email: {
+                          address: 'jondoe@example.com',
+                          subscribeStatus: 'subscribed',
+                          subscribedAt: '2024-07-12T09:58:14+0000',
+                        },
+                      },
+                      createdAt: '2024-07-12T09:58:14+0000',
+                      address: {
+                        city: 'San Miego',
+                      },
+                      first_name: 'John',
+                      last_name: 'Doe',
+                      lastUpdateSource: 'api',
+                      lastModified: '2024-07-12T13:00:49+0000',
+                      cID: '6690fe3655e334xx028xxx',
+                    },
+                    device: {
+                      type: 'computer',
+                      device: false,
+                      platform: false,
+                      browser: false,
+                      robot: true,
+                    },
+                    externalId: [],
+                  },
+                  integrations: {
+                    Cordial: false,
+                  },
+                  type: 'track',
+                  event: 'browse',
+                  originalTimestamp: '2024-07-12T14:45:25+00:00',
+                  properties: {
+                    event_id: '669141857b8cxxx1ba0da2xx',
+                    email: 'jondoe@example.com',
+                    category: 'Shirts',
+                    url: 'http://example.com/shirts',
+                    description: 'A really cool khaki shirt.',
+                    price: 9.99,
+                    title: 'Khaki Shirt',
+                    test_key: 'value',
+                    _id: '669141857b8cxxx1ba0da2xx',
```

Comment:
we already mapping _id to event_id.
Let's remove _id from properties

Reply:
done

Reply:
you have removed `event_id` as well in transformed payload